VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Open an app store to display the app.
+     *
+     * @param context A Context for starting the new Activity.
+     */
+    public static void openAppInAppStore(final Context context) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.addCategory(Intent.CATEGORY_BROWSABLE);
+        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        try {
+            context.startActivity(intent);
+        } catch (ActivityNotFoundException e) {
+            intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
+                            BuildConfig.APPLICATION_ID));
```

Comment:
Let's not solve for apps intercepting the `market://` URI, but lets definitely make both of these configurable. How about just a config for the two URIs (name them whatever you want). E.g.:
```
APP_STORE_NATIVE_URI: 'market://details?id={applicationId}'
APP_STORE_WEB_URI: 'https://play.google.com/store/apps/details?id={applicationId}'
```
We'll try the native URI first, and fill in the `{applicationId}` placeholders, if present.